src/core/Akka.Persistence.Tests/Fsm/PersistentFSMSpec.cs

```diff
@@ -414,6 +414,39 @@ public void PersistentFSM_must_allow_cancelling_stateTimeout_by_issuing_forMax_n
             probe.ExpectNoMsg(1.Seconds());
         }
 
+        [Fact]
+        public void PersistentFSM_must_save_periodical_snapshots_if_enablesnapshotafter()
+        {
+            var sys2 = ActorSystem.Create("PersistentFsmSpec2", ConfigurationFactory.ParseString(@"
+                akka.persistence.fsm.enable-snapshot-after = on
```

Comment:
so is the feature itself not enabled? looks like you added it in the Akka.Persistence.cong